src/controllers/update/update.beneficial.owner.gov.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
Strange to have controller for 2 out of 3. Is plan to add other / maybe separate controllers?